entry.js

```diff
@@ -0,0 +1,49 @@
+import { 
+  buttonCreate, 
+  createTodoDiv, 
+  cancelTodoButton, 
+  textArea, 
+  addTodoButton, 
+  todoWrapper,
+  emptyNotice,
+} from "./components/domElements.js";
+
+import { 
+  createTodoItem,
+  renderTodoList,
+ } from "./components/domActions.js";
+
+import { openCreateTodo, cancelTodo } from "./components/domActions.js";
```

Comment:
They seem to import from the same file. We can use one statement.
```suggestion
import { 
  createTodoItem,
  renderTodoList,
  openCreateTodo,
  cancelTodo
 } from "./components/domActions.js";
```